### PR TITLE
New pipeline to call TDBuild for VSIX Localization

### PR DESCRIPTION
**Pull request by kythant**
New pipeline to call TDBuild for VSIX Localization

This pipeline will run on some determined schedule and will continually update the localized VSIX resources on the branch that it is ran on as they come in.

### CODE REVIEW

File: build/AzurePipelinesTemplates/WindowsAppSDK-CallTouchDownBuild-Steps.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  - 'None'
+  - 'COMMIT'
+
+steps:
+- checkout: self
+  persistCredentials: true
+
+# This is needed to avoid credentials errors for later in the TDBuild Task
+- script: |
+    git config --global user.email "you@example.com"
```
Comment (BenJKuhn) [inline build/AzurePipelinesTemplates/WindowsAppSDK-CallTouchDownBuild-Steps.yml]: Can we put something in here to indicate the commit is coming from the touchdown build?

**Review by BenJKuhn — APPROVED** [review]

**Review by alwu-msft — APPROVED** [review]
:shipit:

**Pull request merged**